www.foia.gov/_api/request-payload/oge.json

```diff
@@ -1,4 +1,6 @@
 {
```

Comment:
Should we rename this file since the contents are generic?

Reply:
I'm going to leave it for now so I don't throw OGE off, but yeah, I was imagining having several different payloads to test against (with/without file attachment, invalid, etc)... But we'll probably have the foia-api part working soon anyway, so we won't need it in the long term.

Reply:
👍